functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -75,6 +76,42 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
 			Expect(profile).To(BeEquivalentTo(expectedProfile), "regression test failed for '%s' case", expectedProfilePath)
 		}
 	})
+
+	It("should describe the cluster from must-gather data in info mode", func() {
+		Expect(ppcPath).To(BeAnExistingFile())
+
+		// directory base name => full path
+		mustGatherDirs := getMustGatherDirs(mustGatherPath)
+
+		for name, path := range mustGatherDirs {
+			cmdArgs := []string{
+				"--info",
+				fmt.Sprintf("--must-gather-dir-path=%s", path),
+			}
+			cmdEnv := map[string]string{
+				"PERFORMANCE_PROFILE_CREATOR_OUTPUT_JSON": "1",
```

Comment:
Why is this equal to 1? I think it would be better to have `PERFORMANCE_PROFILE_CREATOR_OUTPUT_JSON= true`.

Reply:
shorter to type, and doesn't really make a difference since the value is ignored :)

Reply:
anyway, changing to "true"

Reply:
Let move this to UI instead!

Reply:
sure, done!